main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,305 @@
-# Starting a Project
+# Your First Agoric Dapp
+In these steps you will be getting your first Agoric dapp up and running!
 
-After you've [installed the Agoric SDK](./README.md) (_recall: use `agoric --version` to confirm._), you're ready for your first _Agoric Dapp_ (decentralized application).
+![Your first Agoric dapp](./assets/045_small.png)
```

Comment:
rather than having them fetch another image, we can use markup to get this screenshot to use less space